koptional-rxjava2-extensions/src/test/kotlin/com/gojuno/koptional/rxjava2/RxJava2ExtensionsSpec.kt

```diff
@@ -124,7 +124,7 @@ class RxJava2ExtensionsSpec : Spek({
     context("Maybe") {
 
         context("Stream with Some") {
-            val stream by memoized {
+            val stream: Maybe<out Optional<String>> by memoized {
                 Maybe.just(Some("a") as Optional<String>)
```

Comment:
So we should remove `as Optional<String>` then here and in similar places in tests